script.js

```diff
@@ -0,0 +1,35 @@
+
+
+$(document).ready(function(){
+
+  function blogIsBlog(event) {
+    event.preventDefault();
+  }
+
+  function showReadMoreText () {
+    console.log('show hidden paragraph');
+    $('#show-this-on-click').slideDown();
+    $('.readless').show();
+    $('.readmore').hide();
+  }
+
+  function showLessMoreText () {
+    console.log('hide shown paragraph');
+    $('#show-this-on-click').slideUp();
+    $('.readless').hide();
+    $('.readmore').show();
+  }
+
+  function showLearnMore () {
+    console.log('show learn more');
+    $('.learnmore').hide();
+    $('#learnmoretext').slideDown ();
```

Comment:
Remove the white  space between `slideDown` and the `()` or this code won’t work.